src/GitHub.VisualStudio.UI/UI/Controls/InfoPanel.xaml.cs

```diff
@@ -45,6 +45,7 @@ static void UpdateVisibility(DependencyObject d, DependencyPropertyChangedEventA
         void Button_Click(object sender, RoutedEventArgs e)
         {
             this.Visibility = Visibility.Collapsed;
+            this.Message = string.Empty;
```

Comment:
You can actually remove the `this.Visibility = Visibility.Collapsed;` line above and just do this one.
As soon as you set the message to empty, the `UpdateVisibility` method triggers and the visibility will get set to collapsed ([on line 42](https://github.com/github/VisualStudio/pull/457/files#diff-9482f34c5d4f74fce8bf57c770dac73bL42))

Side effects ftw! :smile: